cmd/evmosd/testnet.go

```diff
@@ -95,6 +99,8 @@ func addTestnetFlagsToCmd(cmd *cobra.Command) {
 	cmd.Flags().String(flags.FlagChainID, "", "genesis file chain-id, if left blank will be randomly created")
 	cmd.Flags().String(sdkserver.FlagMinGasPrices, fmt.Sprintf("0.000006%s", cmdcfg.BaseDenom), "Minimum gas prices to accept for transactions; All fees in a tx must meet this minimum (e.g. 0.01photino,0.001stake)")
 	cmd.Flags().String(flags.FlagKeyType, string(hd.EthSecp256k1Type), "Key signing algorithm to generate keys for")
+	cmd.Flags().String(flagBaseFee, "1000000000", "The params base_fee in the feemarket module in geneis")
```

Comment:
BaseFee is a Uint. Is there a particular reason why this is a String?

Also, please use the constant from `"github.com/ethereum/go-ethereum/params"` as default value

```suggestion
	cmd.Flags().Uint64(flagBaseFee, params.InitialBaseFee, "The params base_fee in the feemarket module in geneis")
```

Reply:
No, baseFee is an Int. You can see his proto statement: https://github.com/evmos/evmos/blob/v15.0.0-rc1/proto/ethermint/feemarket/v1/feemarket.proto#L26

And the Uint64 type cannot be used, because baseFee can be a large number, such as 10000000000000gwei, and Uint64 cannot meet its needs at this time.